packages/inngest/src/components/InngestCommHandler.ts

```diff
@@ -824,7 +824,7 @@ export class InngestCommHandler<
       }
 
       if (method === "GET") {
-        const registerBody = this.registerBody(this.reqUrl(url));
+        const registerBody = this.registerBody(this.reqUrl(url), null);
```

Comment:
Can we switch this to named args? It isn't clear what `null` is supposed to be without looking at the function signature